src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * The main service provider for the version 2 of the Widgets.
+ *
+ * @since   TBD
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+/**
+ * Class Service_Provider
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Variable that holds the name of the widgets being created
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $widgets = [
+		'tribe_events_list_widget',
+	];
+
+	/**
+	 * Binds and sets up implementations.
+	 */
+	public function register() {
+		if ( ! tribe_events_views_v2_is_enabled() ) {
+			return;
+		}
+
+		$this->hook();
+	}
+
+	/**
+	 * Function used to attach the hooks associated with this class.
+	 *
+	 * @since TBD
+	 */
+	public function hook() {
+		add_filter( 'tribe_widgets', [ $this, 'add_widgets' ] );
+	}
+
+	/**
+	 *
```

Comment:
:no_entry_sign: Doc comment for parameter "$widgets" missing

<sub>hash: 908fbc84fad4ce012ff74a4915fea672</sub>